src/components/DomainField/DomainField.js

```diff
@@ -18,14 +19,14 @@ function filterSubdomain(subdomain, detectFullDomains) {
 
 const DomainField = React.forwardRef(function DomainField(
   {
+    detectFullDomains = false,
```

Comment:
Are these default values needed with the `defaultProps`?

Reply:
Nope I forgot these, thanks!